geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/StringBytesGlossary.java

```diff
@@ -238,6 +238,12 @@
   @MakeImmutable
   public static final byte[] bRADISH_DUMP_HEADER = stringToBytes("RADISH");
 
+  @MakeImmutable
+  public static final byte[] bRADISH_WITHSCORES = stringToBytes("WITHSCORES");
```

Comment:
There is already a byte array constant for `WITHSCORES` further up in this class (line 180), so this one is not necessary.

Reply:
Good catch, fixed!